snap7/server.py

```diff
@@ -38,7 +38,9 @@ def __init__(self, log=True):
         self.create()
         if log:
             self._set_log_callback()
-    
+        self._callback = None
+        self._read_callback = None
```

Comment:
Is this related to the python3 upgrade?

Reply:
Both has been initiated outside of __init__ so I add them as None (for readability).
While I let a linter run my code, I fixed that on the fly. It would be a better Python style in general.